src/server/game/Entities/Unit/Unit.h

```diff
@@ -1420,12 +1428,16 @@ class Unit : public WorldObject
     [[nodiscard]] uint8 GetLevel() const { return getLevel(); }
     uint8 getLevelForTarget(WorldObject const* /*target*/) const override { return GetLevel(); }
     void SetLevel(uint8 lvl, bool showLevelChange = true);
-    [[nodiscard]] uint8 getRace(bool original = false) const;
-    void setRace(uint8 race);
-    [[nodiscard]] uint32 getRaceMask() const { return 1 << (getRace(true) - 1); }
-    [[nodiscard]] uint8 getClass() const { return GetByteValue(UNIT_FIELD_BYTES_0, 1); }
+    [[nodiscard]] uint8 getRace() const { return GetByteValue(UNIT_FIELD_BYTES_0, UNIT_BYTES_0_OFFSET_RACE); }
+    void setRace(uint8 race) { SetByteValue(UNIT_FIELD_BYTES_0, UNIT_BYTES_0_OFFSET_RACE, race); }
+    [[nodiscard]] uint32 getRaceMask() const { return 1 << (getRace() - 1); }
+    [[nodiscard]] uint8 getClass() const { return GetByteValue(UNIT_FIELD_BYTES_0, UNIT_BYTES_0_OFFSET_CLASS); }
+    void setClass(uint8 classId) { SetByteValue(UNIT_FIELD_BYTES_0, UNIT_BYTES_0_OFFSET_CLASS, classId); }
```

Comment:
For these, I would like if you can create a "duplicate" with correct capitalization so we can move out form lowerUpper.

To not break modules do this
```cpp
void SetRace(uint8 race) { SetByteValue(UNIT_FIELD_BYTES_0, UNIT_BYTES_0_OFFSET_RACE, race); }
void setRace(uint8 race) { SetRace(race); }
```

And then update it all throughout the core ofc

Reply:
Give me in like a week or two when im back into availability mode. This pr not exactly finished.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14843/commits/c5add62567a2f1850552da69f846a7796eb94b28